common/src/main/java/org/apache/nemo/common/coder/IntEncoderFactory.java

```diff
@@ -59,8 +59,7 @@ public String toString() {
    * IntEncoder.
    */
   private final class IntEncoder implements Encoder<Integer> {
-
-    private final DataOutputStream outputStream;
+    private final transient DataOutputStream outputStream;
```

Comment:
Could you erase the transient keyword too?
Let’s just not resolve this issue for now.